README.md

```diff
@@ -123,6 +137,8 @@ A few options are provided for you to allow for some customization
 ### What About Building the Site Before Committing?
 There is no need to build the rendered site before committing and submitting a PR. This will all happen on the CI side to test and build the rendered site.
 
+## Running tests
+Running the tests before committing should be done and can be accomplished by running `yarn test` from the terminal. This will run a series of tests to make sure that everything is working as expected and that your changes did not affect anything that was not planned.
```

Comment:
we should add something like the following:

If you already have your local server running and available at `localhost:8080`, you should only run `yarn test-only` which bypasses the starting of the local server

At times, all of the tests will fail.  This is caused by the selenium suite not shutting down correctly. If this happens, run `yarn stop` and it should resolve itself on the next test run.